<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollageCompleted</class>
 <widget class="QDialog" name="CollageCompleted">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Collage Completed</string>
  </property>
  <widget class="QLabel" name="successfulLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Collages created successfully.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="folderButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Go to the Folder</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
